ci/blackbox/rdsbroker_test.go

```diff
@@ -306,50 +305,77 @@ func openConnection(databaseURI string) (*sql.DB, error) {
 	return sql.Open(dbURL.Scheme, dsn)
 }
 
-func setupPermissionsTest(databaseURI string) error {
+func setupPermissionsTest(databaseURI, serviceID string) error {
 	db, err := openConnection(databaseURI)
 	if err != nil {
 		return err
 	}
 	defer db.Close()
 
-	_, err = db.Exec("CREATE TABLE " + permissionCheckTableName + " (id integer)")
+	_, err = db.Exec("CREATE TABLE permissions_check (id integer)")
 	if err != nil {
 		return fmt.Errorf("Error creating table: %s", err.Error())
 	}
 
-	_, err = db.Exec("INSERT INTO " + permissionCheckTableName + " VALUES(42)")
+	_, err = db.Exec("INSERT INTO permissions_check VALUES(42)")
 	if err != nil {
 		return fmt.Errorf("Error inserting record: %s", err.Error())
 	}
 
+	switch serviceID {
```

Comment:
On closer inspection this isn't actually pulled from the catalog, it's [hard coded](https://github.com/alphagov/paas-rds-broker/blob/b9d587b9090a86224e500786f397932c2664b603/ci/blackbox/rdsbroker_test.go#L124). We should still change it though because pulling it from the URI scheme is much less brittle.